src/SearchButton.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+function SearchButton() {
+  return (SearchButton = <button className="btn btn-primary">Search</button>);
+}
```

Comment:
Ok, in the code above, you are providing something that I think is a callback, via props, and naming the prop `SearchButton`, right?

Where inside the SearchButton are you receiving the props? Where inside the SearchButton are you using the callback you are providing to it?

Reply:
As i remember "className="btn btn-primary" is a provided line of code which i should change from html into JSX but keep the styles. I just kept thing i don't understand for safety